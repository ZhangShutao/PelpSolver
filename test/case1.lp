a | b.
c :- K[0.5, 1]a.