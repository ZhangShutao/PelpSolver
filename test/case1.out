WorldView 1:
{[0.5]:{a,c},[0.5]:{b,c}}